### PR TITLE
Quick Board Tips blog

### PR DESCRIPTION
**Pull request by wpschaub**
Quick blog post in liu of cookbook sync meeting. Expose more hidden gems of the Azure Boards feature.

### CODE REVIEW

File: content/azure-boards-tips-retain-hierarchy-with-filter.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+---
+
+# Flip a toggle to get the view you want
+
+A feature I discovered is the "keep hierarchy with filters" toggle under **View Options**. 
+
+> ![Hierarchy Flag](../images/azure-boards-tips-retain-hierarchy-with-filter-3.png) 
+
+If you toggle that to **On** you get the view I believe should be the default. Best pf both worlds ... filter set, and parent and child hierarchy enabled q;-)
```
Comment (MerlinDarkstar) [inline content/azure-boards-tips-retain-hierarchy-with-filter.md]: typo - Best pf both
s/b - Best of both

Comment (MerlinDarkstar) [reply content/azure-boards-tips-retain-hierarchy-with-filter.md]: I think the 'q' at the end is also a typo - I seem to add those once in a while when doing the character emoji ;-)

**Review by MerlinDarkstar — COMMENTED** [review]
very minor typos - but great article / info to share!!

File: content/azure-boards-tips-retain-hierarchy-with-filter.md (hunk 1 above)
Comment (wpschaub) [reply content/azure-boards-tips-retain-hierarchy-with-filter.md]: Fixed

Comment (wpschaub) [reply content/azure-boards-tips-retain-hierarchy-with-filter.md]: Removed the emoji. 'q' is the smiley's baseball hat :)

**Review by danielrbroderick — APPROVED** [review]

**Review by MerlinDarkstar — APPROVED** [review]

**Pull request merged**